docs/reference/generated/menu-radio-group.json

```diff
@@ -1,6 +1,5 @@
 {
   "name": "MenuRadioGroup",
-  "description": "Groups related radio items.\nRenders a `<div>` element.",
```

Comment:
The description is missing here.

Reply:
I don't know if it is related, but the JSdocs on this component misses the "Documentation" information.

Reply:
MenuRadioGroup is not what is directly exported - it's `const MemoizedMenuRadioGroup = React.memo(MenuRadioGroup);` which doesn't have any description.